doc/developer-guides/hld/hostbridge-virt-hld.rst

```diff
@@ -0,0 +1,43 @@
+.. _hostbridge_virt_hld:
+
+Hostbridge emulation
+####################
+
+Overview
+********
+
+Hostbridge emulation is based on PCI emulation. However hostbridge emulation only sets PCI configuration space. Device model set the PCI configuration space for hostbridge in SOS, then expose to UOS to detect the PCI hostbridge.
+
+PCI Host Bridge and hierarchy
+*****************************
+
+There is PCI host bridge emulation in DM. The bus hierarchy is determined by acrn-dm command line input. Using this command line, as an example::
+
+        acrn-dm -A -m 2048M -c 3 \
+        -s 0:0,hostbridge \
+        -s 1:0,lpc -l com1,stdio \
+        -s 5,virtio-console,@pty:pty_port \
+        -s 3,virtio-blk,b,/data/clearlinux/clearlinux.img \
+        -s 4,virtio-net,tap_LaaG --vsbl /usr/share/acrn/bios/VSBL.bin \
```

Comment:
I would prefer if we could use a script used for the NUCs as an example (so using OVMF.fd, etc.)